application-templates/starter-typescript/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "@commercetools-frontend/tsconfig",
+  "include": ["./jest.test.setup.ts"]
```

Comment:
I am including it here for the time being, as I am not sure if:
- we need that at all 
- or perhaps this file belongs in the `@types` folder